github/src/main/scala/org/typelevel/sbt/TypelevelGitHubPlugin.scala

```diff
@@ -82,19 +82,24 @@ object TypelevelGitHubPlugin extends AutoPlugin {
       val userRepo = gitHubUserRepo.value
       val infoOpt = scmInfo.value
 
-      if (tlIsScala3.value)
-        tagOrHash.toSeq flatMap { vh =>
-          userRepo.toSeq flatMap {
-            case (user, repo) => Seq(s"-source-links:github://${user}/${repo}", "-revision", vh)
-          }
-        }
-      else
-        tagOrHash.toSeq flatMap { vh =>
-          infoOpt.toSeq flatMap { info =>
-            val path = s"${info.browseUrl}/blob/${vh}€{FILE_PATH}.scala"
-            Seq("-doc-source-url", path)
-          }
+      tagOrHash.toSeq flatMap { vh =>
+        scalaVersion.value match {
+          case V(V(3, _, _, _)) =>
+            userRepo.toSeq flatMap {
+              case (user, repo) =>
+                Seq(s"-source-links:github://${user}/${repo}", "-revision", vh)
+            }
+          case (V(V(2, minor, patch, _))) =>
+            infoOpt.toSeq flatMap { info =>
+              val path =
+                // see https://github.com/scala/bug/issues/12867#issuecomment-1718481858
+                if (minor > 13 || minor == 13 && patch.exists(_ >= 12))
```

Comment:
The behaviour here is to assume that someone running on `2.13` (whatever that means), would want the mitigation off

```suggestion
                if (minor > 13 || minor == 13 && patch.forall(_ >= 12))
```

This would instead assume that a `2.13`er would instead be running the latest version: is this what we want?

Reply:
Based on https://github.com/typelevel/sbt-typelevel/pull/643#discussion_r1327523924, I think we should fallback to adding the slash.